docs-2.0/3.ngql-guide/1.nGQL-overview/3.graph-patterns.md

```diff
@@ -54,6 +54,10 @@ Patterns can also describe a vertex that has multiple tags. For example:
 (a:User:Admin)-[]->(b)
 ```
 
+!!! compatibility "OpenCypher compatibility"
+
+    The `MATCH` statement in nGQL does not support matching multiple tags with `(a:User:Admin)`. If you need to match multiple tags, use filtering conditions, such as `WHERE "User" IN tags(n) AND "Admin" IN tags(n)`.
```

Comment:
it  might change in 2.5.0. To be confirmed.

Reply:
This change is made according to the 2.5.0 release.